.travis.yml

```diff
@@ -12,11 +12,11 @@ compiler:
 notifications:
     email:
         recipients:
-            - su2code-dev@lists.stanford.edu
+            - tobias.kattmann@de.bosch.com
```

Comment:
Please revert the email and branch in .travis.yml

Reply:
I'll do that with the next commit in which I change the affected regression test values.